### PR TITLE
Relax JSON dependency to allow 2.x

### PR DESCRIPTION
**Pull request by isuda**
Simplecov also changed this a while ago:

https://github.com/colszowka/simplecov/commit/b47a3bd33d70f90ea2339ec2cef2c7427a6cb825

### CODE REVIEW

File: simplecov-cobertura.gemspec [hunk 1]
```diff
@@ -26,5 +26,5 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'nokogiri', '~> 1.6'
 
   spec.add_dependency 'simplecov', '~> 0.8'
-  spec.add_dependency 'json', '~> 1.8'
+  spec.add_dependency "json", ">= 1.8", "< 3"
```
Comment (denispasin) [inline simplecov-cobertura.gemspec]: Do we need the dependency at all ? simplecov already have it.

**Comment by jesperronn** [comment]
Until this is merged, I cannot update my application to Ruby 2.4 since Ruby 2.4 defaults json to 2.0.4 (I'm using Ruby 2.4.2)

**Comment by jesperronn** [comment]
The build (run 94) is failing with this message:

```
  bundle install --gemfile=$SHIPPABLE_GEMFILE $SHIPPABLE_BUNDLER_ARGS 2s  
Fetching gem metadata from https://rubygems.org/.............
Fetching version metadata from https://rubygems.org/.
nokogiri-1.8.1 requires ruby version >= 2.1.0, which is incompatible with the
current version, ruby 1.9.3p551
```

So, if simplecov-cobertura depends on nokogiri, the 1.9 dependencies should probably be handled in a separate gemfile?

**Comment by jessebs** [comment]
I should have time this weekend or coming week to merge it in.  I'll also evaluate the comment from  @denispasin as to whether we even need the dependency.

**Comment by denispasin** [comment]
I tested it there ( https://github.com/denispasin/simplecov-cobertura/blob/master/simplecov-cobertura.gemspec ) and it worked well

**Comment by jessebs** [comment]
I took the discussion points here and created a new pull request #6

**Pull request closed without merging**